build_tools/bazel/workspace.bzl

```diff
@@ -8,6 +8,63 @@
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 load("@bazel_skylib//lib:paths.bzl", "paths")
 
+CUDA_TOOLKIT_ROOT_ENV_KEY = "IREE_CUDA_TOOLKIT_ROOT"
+
+# See base docker image: We place a stripped down CUDA directory tree
+# here if this env var is set on CI.
+CUDA_DEPS_DIR_FOR_CI_ENV_KEY = "IREE_CUDA_DEPS_DIR"
+
+def cuda_auto_configure_impl(repository_ctx):
+    env = repository_ctx.os.environ
+    found = False
```

Comment:
Need to update the second usage on line 51: `"%ENABLED%": "True" if found else "False"`. Could also create the variable based on this (instead of previously where it got set in both branches of the conditional): `found = cuda_toolkit_root != None`